### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/Convex/Radon.lean [hunk 1]
```diff
@@ -145,7 +143,7 @@ theorem helly_theorem {F : ι → Set E} {s : Finset ι}
   apply helly_theorem' h_convex
   intro I hI_ss hI_card
   obtain ⟨J, hI_ss_J, hJ_ss, hJ_card⟩ := exists_subsuperset_card_eq hI_ss hI_card h_card
-  apply Set.Nonempty.mono <| biInter_mono hI_ss_J (by intro _ _; rfl)
+  apply Set.Nonempty.mono <| biInter_mono hI_ss_J (fun _ _ => Set.Subset.rfl)
```
Comment (grunweg) [inline Mathlib/Analysis/Convex/Radon.lean:146]: Can you use "\maps", please?

File: Mathlib/Combinatorics/SimpleGraph/Matching.lean [hunk 2]
```diff
@@ -23,7 +23,6 @@ one edge, and the edges of the subgraph represent the paired vertices.
 
 * `SimpleGraph.Subgraph.IsMatching`: `M.IsMatching` means that `M` is a matching of its
   underlying graph.
-  denoted `M.is_matching`.
```
Comment (grunweg) [inline Mathlib/Combinatorics/SimpleGraph/Matching.lean:26]: Nice find!

File: Mathlib/GroupTheory/Perm/List.lean [hunk 3]
```diff
@@ -155,7 +155,7 @@ theorem formPerm_apply_getLast (x : α) (xs : List α) :
 @[simp]
 theorem formPerm_apply_getElem_length (x : α) (xs : List α) :
     formPerm (x :: xs) ((x :: xs)[xs.length]) = x := by
-  rw [getElem_cons_length, formPerm_apply_getLast]; rfl;
+  rw [getElem_cons_length _ _ _ rfl, formPerm_apply_getLast]
```
Comment (grunweg) [inline Mathlib/GroupTheory/Perm/List.lean:158]: Optional:
```suggestion
  rw [getElem_cons_length (h := rfl), formPerm_apply_getLast]

**Review by grunweg — COMMENTED** [review]
Thanks for the PR - as usual, lots of nice small clean-ups. I have two minor suggestions - feel free to maintainer merge on my behalf once you considered them.

**Comment by Ruben-VandeVelde** [comment]
maintainer merge

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**